src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
 
   }
+  if (a == null)
+   a.charAt(123);
```

Comment:
**Rule**: NP_ALWAYS_NULL

Null pointer dereference&#10;&#10;&#10;&lt;p&gt; A null pointer is dereferenced here.&amp;nbsp; This will lead to a&#10;&lt;code&gt;NullPointerException&lt;/code&gt; when the code is executed.&lt;/p&gt;&#10;

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1116723583>*